src/ralph/data_importer/mixins.py

```diff
@@ -44,5 +88,20 @@ def export_field(self, field, obj):
             Value to export
         """
         if isinstance(field.widget, widgets.ForeignKeyWidget):
-            return str(getattr(obj, self.get_field_name(field), None))
+            value = getattr(obj, self.get_field_name(field), None)
+            return str(value) if value else ''
         return super().export_field(field, obj)
+
+    def get_field_name(self, field):
+        """
+        Returns field name for given field.
+        """
+        for field_name, f in self.fields.items():
+            if field_name.endswith('_str'):
```

Comment:
are you sure this is working correctly? in this first if any information about `field` passed to this function is used